.circleci/config.yml

```diff
@@ -230,6 +230,17 @@ jobs:
     environment:
       BROWSER: << parameters.browser >>
       BVER: << parameters.bver >>
+    trigger-qe-tests:
+      docker:
+        - image: circleci/node:latest
+      steps:
+        - run:
+            name: Trigger QE tests
+            command: |
+              curl --silent --output /dev/null -u USER_TOKEN: -X POST \
+              --header "Content-Type: application/json" \
+              -d '{"branch":"v${CIRCLE_TAG:0:1}","parameters":{"sdk_version": CIRCLE_TAG,"is_rc":true}}' \
+              $SDKS_QE_CIRCLECI_VOICE_JS_TESTS_ENDPOINT
```

Comment:
Please change the name from `SDKS_QE_CIRCLECI_VOICE_JS_TESTS_ENDPOINT` -> `SDKS_QE_CIRCLECI_VIDEO_JS_TESTS_ENDPOINT`. You also need to add this env variable to `sdksqe` context. This url should be pointing to the video qe slave repo.
ex: For voice it is https://circleci.com/api/v2/project/gh/twilio/sdks-qe-voice-js-orchestra-slave/pipeline